### PR TITLE
Comments: commented the function of filter.js and its purpose.

### PR DESCRIPTION
**Pull request by davidArinder**
#### Problem
We need to know what the filters are doing.

#### Solution
Explain what the filters are doing.

#### Testing
Make sure there are comments there.

#### Notes
Added comments.

#4

### CODE REVIEW

File: app/filters.js [hunk 1]
```diff
@@ -4,9 +4,12 @@ const filters = {
   date: Vue.filter('date', (date) => {
     // undef first load
     if (!date) return;
+    // Purpose: remove the time zone.
+    // Converts date into a string, splits the string at the spaces, splice
+    // keeps items 0-5 (removes the time zone), rejoins at the spaces into a
```
Comment (aaronArinder) [inline app/filters.js]: Why have `remove time zone` twice? You get the purpose when you state the purpose, so why include it here?

Comment (davidArinder) [reply app/filters.js]: We should be extremely clear about what the purpose is. Thus, listing it as the purpose. Then in the description of what's actually happening, we list where the purpose is fulfilled.

Comment (aaronArinder) [reply app/filters.js]: Fair enough, dude.

Comment (davidArinder) [reply app/filters.js]: Thanks, bro.

**Pull request merged**